overlord/snapstate/autorefresh.go

```diff
@@ -415,64 +423,74 @@ func (m *autoRefresh) ensureLastRefreshAnchor() {
 	}
 }
 
+func readRefreshScheduleConf(st *state.State) (confStr string, legacy bool, err error) {
```

Comment:
(nitpick^9) given that we call `tr.Get()` maybe this shoud also be `getRefreshScheduleConf()` ?